src/features/TenantSupport/TenantSupport.tsx

```diff
@@ -39,13 +41,17 @@ const issues = [
   },
   {
     category: 'Maintenance Issue',
+    backendCategory: 'maintenance',
+
     options: [
       { value: 'Option 3', label: 'Leaking Faucet', priority: 'med' },
       { value: 'Option 4', label: 'Broken Heater/AC', priority: 'high' },
     ],
   },
   {
     category: 'Building/Common Area Issue',
+    backendCategory: 'building_issues',
+
```

Comment:
Question: wonder how this extra line ended up here? Should have been caught w linting